app/dashboard/views.py

```diff
@@ -415,25 +422,26 @@ def extend_expiration(request, bounty_id):
     }, status=200)
 
 
-@csrf_exempt
+# @csrf_exempt
```

Comment:
@SaptakS why was this disabled in the first place?  any idea ?
@octavioamu could we remove it ?